textBasedAdventure.js

```diff
@@ -1,5 +1,88 @@
+
+const search = (ans) => {
+    let userInput = ans;
+    if (userInput === "defuse the bomb") {
+        console.log("You have no background in Explosive Ordnance Disposal. You set the bomb off and everyone dies!")
+        play();
+    } else if (userInput === "call for back-up") {
+        console.log("Homeland Security assists on call with instructions to defuse the bomb. It's a success! The bomb is now deactivated! The plane makes an emergency landing and an escort arrives to continue the transport. Being a watchful eye, you notice two unmarked vehicles following the convoy.")
+        userInput = readline.question("What do you do?")
+    }
+}
+search();
+
+const fight = (something) => {
+    let userInput = something;
+    if (userInput === "disarm") {
+        console.log("There is a tremendous tussle between the two of you but you are able to secure the prisoner with zipties to his seat.");
+        userInput = readline.question("What do you do next? ease panic or search for bomb");
+        
+    } else if (userInput === "surrender") {
+        console.log("There is no escape for you. The bomb goes off and everyone dies!")
+        play();
+    }
+}
+
+const panic = (something) => {
+    let userInput = something
+    if (userInput === "ease panic") {
+        console.log("Plane stabilizes.")
+        let ans = readline.question("Do you...[search] for bomb or get [prisoner]?")
+        if (ans === "search") {
+            console.log("You locate the bomb in the cargo compartment. You have 30 minutes before detonation.")
+            let answer = readline.question("Do you...defuse the bomb or call for back-up?");
+            search(answer);
+        } else {
+            console.log("Prisoner jumps you from the bathroom and manages to steal your firearm.");
+            userInput = readline.question("Do you...[disarm] the prisoner or [surrender]?");
+            fight();
+        }
+    } else if (userInput === "prisoner") {
+        console.log("Prisoner jumps you from the bathroom and manages to steal your firearm.");
+        userInput = readline.question("Do you...disarm the prisoner or surrender?");
+        fight();
+    }
+ }
+
+
+
+const bomb = (something) => {
+    let userInput = readline.question("What do you do? [alert] the passengers or [search] for bomb?");
+    if (userInput === "alert") {
+        console.log("Panic breaks out in the cabin and the plane becomes unstable. In the commotion, your prisoner escapes to the bathroom.")
+        let ans = readline.question("Do you...ease panic or get prisoner?")
+        panic(ans);
+    } else if (userInput === "search") {
+        console.log("You locate the bomb in the cargo compartment. You have 30 minutes before detonation.")
+        let answer = readline.question("Do you...defuse the bomb or call for back-up?")
```

Comment:
You have the same code here twice, this could have been made into a function and just re-used if need be.